src/wave_generation/IrregularWaveGenerator.cpp

```diff
@@ -15,11 +15,14 @@
 
 #include "ibamr/IrregularWaveGenerator.h"
 #include "ibamr/RNG.h"
-#include "ibamr/app_namespaces.h"
```

Comment:
A major problem with IWYU pragmas is that its really hard to remember to put them everywhere we need them.